home/models/event.py

```diff
@@ -61,7 +61,7 @@ class Event(ClusterableModel):
         related_name="events",
         on_delete=models.SET_NULL,
     )
-
+    video_link = models.URLField(blank=True, null=True)
```

Comment:
I think it's best if we don't use `null=True` and let the empty string count as the false-y value. We don't have many events, so locking the table to set existing rows to `""` won't be painful.